utils/ecc-sig-verifier/Cargo.toml

```diff
@@ -11,9 +11,9 @@ overflow-checks = true
 [dependencies]
 solana-sdk = "1.14.11"
 rocket = { version = "0.5.0-rc.2", features = ["json"] }
-helium-crypto = "0.6.8"
+helium-crypto = "0.7.1"
 hex = "0.4.3"
 serde = "1.0.157"
 bincode = "1.3.3"
-anchor-lang = { workspace = true }
+anchor-lang = { version = "0.26.0", features = ["init-if-needed"] }
```

Comment:
`init-if-needed` feature is only for programs, not needed client side

Reply:
Fixed in [0e38d1f](https://github.com/helium/helium-program-library/pull/316/commits/0e38d1f6ba3521225e8444d72c1a108238c5acc3)